src/opnsense/mvc/app/models/OPNsense/Interfaces/Vip.php

```diff
@@ -42,7 +42,6 @@ public function performValidation($validateFullModel = false)
     {
         $messages = parent::performValidation($validateFullModel);
 
-        $unqiue_addrs = [];
```

Comment:
the var exists as `$unique_addrs` so it would make more sense to fix that typo instead of removing.

Reply:
True! I didn't notice that. Will restore and fix.